src/plugins/audiobridge-plugin.js

```diff
@@ -296,6 +299,13 @@ class AudioBridgeHandle extends Handle {
             janode_event.event = PLUGIN_EVENT.PEER_LEAVING;
             break;
           }
+          /* Room muted event */
+          if (typeof message_data.muted !== 'undefined') {
+            janode_event.data.room = message_data.room;
```

Comment:
This is not necessary, the message handler already fills the `room` property before parsing the events